### ISSUE TITLE
Unable to build standalone or use as dependency

### ISSUE THREAD

**Issue by jsavell** [jupnp/jupnp#125]
I'm interested in using this library for a project, but am having trouble with the maven build process as described in the readme.

The build fails trying to resolve dependencies around the equinox p2 library.
```
[INFO] {osgi.os=linux, osgi.ws=gtk, org.eclipse.update.install.features=true, osgi.arch=x86_64}
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: p2repo 2.5.1
[ERROR]   Missing requirement: org.jupnp 2.5.1 requires 'java.package; javax.servlet 0.0.0' but it could not be found
[ERROR]   Cannot satisfy dependency: org.jupnp.feature.feature.group 2.5.1 depends on: org.eclipse.equinox.p2.iu; org.jupnp 0.0.0
[ERROR]   Cannot satisfy dependency: p2repo 2.5.1 depends on: org.eclipse.equinox.p2.iu; org.jupnp.feature.feature.group [2.5.1,2.5.2)
```
I've tried adding various equinox dependencies manually, but haven't had any success.

Also, when I try using the library as a dependency in my project it builds, but when it actually tries to execute upnp code, an NPE is generated:
```
java.lang.NullPointerException: null
	at org.jupnp.UpnpServiceImpl.startup(UpnpServiceImpl.java:251) ~[org.jupnp-2.5.2.jar:na]
```
That line suggested a problem with the default configuration being used, so I experimented with providing one to the UpnpServiceImpl, as described in the documentation, but have had no luck.

My environment as Maven sees it:
```
Apache Maven 3.6.3
Maven home: /usr/share/maven
Java version: 11.0.8, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.4.0-48-generic", arch: "amd64", family: "unix"
```

Are there any suggestions as to what could be the problem?

**Comment by kaikreuzer** [comment]
@jsavell I can confirm the issue and wasn't able to find an easy fix for it. It seems the Eclipse Orbit update site has changed which broke our build here.

I've decided to completely remove the jUPnP update site as I think it isn't really use anymore - publishing the artefacts on Maven Central should just be fine from now on.

**Comment by kaikreuzer** [comment]
Please also note that the project requires Java 8 to build (while it runs nicely on Java11 as well). I have added a comment in the README on this as well.

**Comment by jsavell** [comment]
Thanks for updating. This did fix the build for me. And I was compiling with both 1.8 and 11 while experimenting. I should have mentioned that.

I still had the NPE when trying to use jupnp as a dependency, though, whether I used 2.5.2 off Maven Central or a locally installed 2.6.0-SNAPSHOT. 

The following are the steps I took to solve the issue. I'm not sure whether the problem is relatively unique to my environment or more broadly applicable. Either way, I'm just providing the steps in case they're useful.

My initial instinct with the no argument Constructor for UpnpServiceImpl appeared to be correct. I could avoid the NPE by passing a new instance of the default configuration, rather than relying on UpnpServiceImpl to generate it internally: 
`new UpnpServiceImpl(new DefaultUpnpServiceConfiguration())
`
With that issue cleared, I then had problems with classdefs not found regarding the servlet-api when this class went to work:
https://github.com/jupnp/jupnp/blob/master/bundles/org.jupnp/src/main/java/org/jupnp/transport/impl/jetty/JettyTransportConfiguration.java

I was able to get past that by explicitly declaring jetty-servlet, jetty-server, and jetty-client in my project POM.

With those steps complete, I was able to successfully start up a UpnpService, execute a search, and output a list of the resulting devices.

Very exciting. Thank you for this project.
